### ISSUE TITLE
Typo in JDK Version

### ISSUE THREAD

**Issue by iSoftech** [spring-projects/spring-security-kerberos#253]
Is the version mentioned in this line is the right version or the one mentioned in the title i.e. JDK17? The below line states jdk1.7.0.

https://github.com/spring-projects/spring-security-kerberos/blame/e07f0036dfcc6c105edfba84bdef4ce543040169/README.adoc#L29

**Comment by dodgex** [comment]
You linked even with the blame... there you can see, that this line was added 9 years ago. So I am sure, it was intended to have 1.7.0 (aka Java 7) mentioned.

I think the readme is (partially?) as deprecated as the [https://github.com/spring-projects/spring-security-kerberos/blob/main/CONTRIBUTING.md](CONTRIBUTING.md) is that links to JIRA that is long gone.

**Comment by iSoftech** [comment]
Yes, I agree, but the title says JDK17 which makes me wonder if it still supports the older version of JDK or it expects latest packages like Spring 6+ / SpringBoot 3+ and JDK17+.

**Comment by dodgex** [comment]
According to [this file](https://github.com/spring-projects/spring-security-kerberos/blob/main/gradle/libs.versions.toml) it seems that the the latest version requires Spring 6.1+ and Spring Security 6.3+. At least it builds against these versions.

**Comment by rwinch** [comment]
Thanks for the report. This is now updated in main
